connect-inject/handler.go

```diff
@@ -198,15 +198,15 @@ func (h *Handler) Handle(_ context.Context, req admission.Request) admission.Res
 	// the Envoy configuration.
 	initContainer, err := h.containerInit(pod, req.Namespace)
 	if err != nil {
```

Comment:
since we're changing it, I wonder if we should add an `Info` level log line saying the name/namespace of the pod it's mutating, maybe on line 176?

Reply:
Done!! 😺